person/models.py

```diff
@@ -216,11 +216,11 @@ class GodFather(Person):
     payment_method = models.CharField(
         max_length=50, choices=PAYMENT_METHOD, verbose_name='Método de pago',)
     bank_account_number = IBANField(
-        include_countries=IBAN_SEPA_COUNTRIES, verbose_name='Número de cuenta bancaria')
+        include_countries=IBAN_SEPA_COUNTRIES,blank=True, null=True)
```

Comment:
I think you should leave the verbose name fields, because if the use the admin panel they will display in English without it: 
![image](https://user-images.githubusercontent.com/64160615/231244026-4f0aae53-0a63-4688-94b1-3f6fc223fbc6.png)